services/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -760,13 +760,17 @@ public void onComplete(Result result)
         return;
       }
 
-      boolean success = result.isSucceeded();
```

Comment:
Why did we change this logic of `success` from `result.isSucceeded()` to ` result.getResponse().getStatus()`?

Is this related to the error code change?

Reply:
Ah yes makes sense. Could you replace the `success && result.getResponse().getStatus() == Status.OK.getStatusCode()` you've referenced with simply `success` now (since the logic is effectively the same)?